### ISSUE TITLE
Add support for tslint 6.x

### ISSUE THREAD

**Issue by rooby** [ReactiveX/rxjs-tslint#72]
Currently the peer dependency of tslint: ^5.0.0 means you get this warning when using tslint 6.x

> npm WARN rxjs-tslint@0.1.8 requires a peer of tslint@^5.0.0 but none is installed. You must install peer dependencies yourself.

Can support for 6.x be added?

**Comment by joshuakeel** [comment]
I just ran into this as well, when updating to Angular 10.

**Comment by joboyx** [comment]
Same with me, I've encountered this when running Angular v11 upgrade:
```
$ ng update @angular/core @angular/cli
Using package manager: 'npm'
Collecting installed dependencies...
Found 66 dependencies.
Fetching dependency metadata from registry...
                  Package "rxjs-tslint" has an incompatible peer dependency to "tslint" (requires "^5.0.0", would install "6.1.3").
✖ Migration failed: Incompatible peer dependencies found.
Peer dependency warnings when installing dependencies means that those dependencies might not work correctly together.
You can use the '--force' option to ignore incompatible peer dependencies and instead address these warnings later.
  See "/tmp/ng-ABeah5/angular-errors.log" for further details.
```
have you tried to force (`--force`) and ignore the peer dependency warning? Would there be issues?

**Comment by rooby** [comment]
@rooby Yep, I'm currently doing that and haven't had any issues.

**Comment by madhukudala** [comment]
![image](https://user-images.githubusercontent.com/7322186/124151702-752e4d00-dab0-11eb-89b1-6262735f6b15.png)

I got the same issue. Because of this issue, I am unable to update the angular version from 10 to 11. Is there any solution to fix this?
